src/BorrowBookUI.java

```diff
@@ -85,12 +85,12 @@ public void run() {
 					
 				
 			case FINALISING:
-				String Ans = input("Commit loans? (Y/N): ");
+				String Ans = input("Commit loans? (Y/N): "); //was Ans
```

Comment:
I'm sure "Ans" is a variable, so it should be changed to "ans".

Reply:
great! has been now updated